bin/pullRawDataFromDS.sh

```diff
@@ -249,13 +249,17 @@ else
 	done
 fi
 
-#
-# Cleanup old data if data transfer with rsync finished successfully (and hence did not crash this script).
-#
-log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "Deleting data older than 14 days from ${HOSTNAME_DATA_STAGING%%.*}:/groups/${GROUP}/${SCR_LFS}/ ..."
-# shellcheck disable=SC2029
-/usr/bin/ssh "${HOSTNAME_DATA_STAGING}" "/bin/find /groups/${GROUP}/${SCR_LFS}/ -mtime +14 -ignore_readdir_race -delete"
-
+if [[ "${GROUP}" == "umcg-gst" ]]
```

Comment:
Dit is misschien wel een korte klap, maar zorgt voor een hard-coded groepsnaam in het script. Had wel of niet data opruimen niet gekund op basis van een variabele in de ```${group}.cfg```?

Reply:
check